projects/swimlane/ngx-ui/src/lib/components/json-editor/json-editor-flat/json-editor-node-flat/json-editor-node-flat.component.scss

```diff
@@ -1,6 +1,13 @@
 @import 'colors/variables';
 @import '../../json-editor.extensions.scss';
-
+.code-editor {
```

Comment:
We can put both changes together

Reply:
Hey you are right , just made the change, thanks.